pkg/controllers/monitoring/monitoring-stack/conditions.go

```diff
@@ -9,36 +9,45 @@ import (
 )
 
 const (
-	AvailableReason                = "MonitoringStackAvailable"
-	ReconciledReason               = "MonitoringStackReconciled"
-	FailedToReconcileReason        = "FailedToReconcile"
-	PrometheusNotAvailable         = "PrometheusNotAvailable"
-	PrometheusNotReconciled        = "PrometheusNotReconciled"
-	PrometheusDegraded             = "PrometheusDegraded"
-	ResourceSelectorIsNil          = "ResourceSelectorNil"
-	CannotReadPrometheusConditions = "Cannot read Prometheus status conditions"
-	AvailableMessage               = "Monitoring Stack is available"
-	SuccessfullyReconciledMessage  = "Monitoring Stack is successfully reconciled"
-	ResourceSelectorIsNilMessage   = "No resources will be discovered, ResourceSelector is nil"
-	ResourceDiscoveryOnMessage     = "Resource discovery is operational"
-	NoReason                       = "None"
+	AlertmanagerNotAvailable         = "AlertmanagerNotAvailable"
+	AlertmanagerDegraded             = "AlertmanagerDegraded"
+	AlertmanagerNotReconciled        = "AlertmanagerNotReconciled"
+	AvailableReason                  = "MonitoringStackAvailable"
+	ReconciledReason                 = "MonitoringStackReconciled"
+	FailedToReconcileReason          = "FailedToReconcile"
+	PrometheusNotAvailable           = "PrometheusNotAvailable"
+	PrometheusNotReconciled          = "PrometheusNotReconciled"
+	PrometheusDegraded               = "PrometheusDegraded"
+	ResourceSelectorIsNil            = "ResourceSelectorNil"
+	CannotReadPrometheusConditions   = "Cannot read Prometheus status conditions"
+	CannotReadAlertmanagerConditions = "Cannot read Alertmanager status conditions"
+	AvailableMessage                 = "Monitoring Stack is available"
+	SuccessfullyReconciledMessage    = "Monitoring Stack is successfully reconciled"
+	ResourceSelectorIsNilMessage     = "No resources will be discovered, ResourceSelector is nil"
+	ResourceDiscoveryOnMessage       = "Resource discovery is operational"
+	NoReason                         = "None"
 )
 
-func updateConditions(ms *v1alpha1.MonitoringStack, prom monv1.Prometheus, recError error) []v1alpha1.Condition {
+func updateConditions(ms *v1alpha1.MonitoringStack, prom monv1.Prometheus, alertManager *monv1.Alertmanager, recError error) []v1alpha1.Condition {
```

Comment:
Lets keep the convention the same as `monv1.Prometheus` argument

```suggestion
func updateConditions(ms *v1alpha1.MonitoringStack, prom monv1.Prometheus, alertmanager monv1.Alertmanager, recError error) []v1alpha1.Condition {
```